### ISSUE TITLE
How do I generate & export (as a video?) visualization like `A Day's Worth of Trips in London` given a csv file with data?

### ISSUE THREAD

**Issue by alexkokorin2** [uber/kepler.gl#104]
[Video](https://www.youtube.com/watch?v=OwwG865Qjh0)

**Comment by wasauce** [comment]
@alexkokorin2 Did you learn how? I am also interested

**Comment by alexkokorin2** [comment]
Nope, the idea is that it _should_ be [possible](https://eng.uber.com/keplergl/):

> visualize it with different map layers, explore it by filtering and aggregating it, and eventually export the final visualization as a static map or an animated video.

**Comment by heshan0131** [comment]
@wasauce, The video was not made with kepler.gl, it was made a long time ago before we even had kepler.gl. I built an app with deck.gl's trip layer, exported frames by setting up an express server then calling canvas.toDataURL on every render. Edited the frames in AfterEffects to add the focus lens blur effect. 

However, we are going to add deck.gl's trip layer to kepler.gl soon, making a similar video like this will soon be possible with kepler.gl. Stay tuned!

**Comment by wasauce** [comment]
@heshan0131 Thank you for the reply! I will stay tuned (already watching the repo).

This is really impressive work and a great tool. I really appreciate it. I am working on a blog post looking at scooter data heavily using this tool that I aim to post this week. Thanks again.
